Fix pages structure

Aichnerc opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**

*See #2 for current behavior*


**What is the new behavior (if this is a feature change)?**
- The folder- and filestructure of components/pages has been fixed to maintain the standard

Aichnerc committed:
Fix pages structure

The folder- and filestructure of components/pages
has been fixed to maintain the standard.

kleberbaum commented on src/index.scss:
```diff
@@ -5,7 +5,7 @@
 @import 'utilities/base';
 
 // Local
```
NOTE: I know this comment is not new, but It's kinda more a headline than a comment.

Aichnerc replied:
Thank you for your notice. This issue has been taken care of.

kleberbaum reviewed (changes requested):
It should be k to merge, but I don't like the Local comment.

Aichnerc committed:
Improve comments

The quality of the comments has been
improved in the index.scss file due
to a request of @kleberbaum.
Ref: #3

kleberbaum commented on src/index.scss:
```diff
@@ -1,10 +1,10 @@
 // Utilities
```
NOTE: This comment is also more a headline than a comment.

kleberbaum reviewed (changes requested):
It would be ok to merge, but I found another comment I don't like.

Aichnerc committed:
Improve comments
The quality of the comments has been
improved in the index.scss file due
to a request of @kleberbaum.
Ref: #3

kleberbaum reviewed (approved):
Is ready to merge. ^^
